piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java

```diff
@@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
     }
 
     /**
+     * Configure Piwik to receive more than 5 custom variable
+     * http://piwik.org/faq/how-to/faq_17931/
+     * Changing this limit will affect all Tracker instances
+     * @param maxVariables
+     */
+    public void setMaxCustomVariables(int maxVariables){
+        CustomVariables.MAX_VARIABLES = maxVariables;
```

Comment:
I think the most elegant way will be to remove limit of custom vars completely. 
We just can assume that developers are responsible enough and read the doc strings.